### PR TITLE
PMM-9211 Mysql backup tests

### PR DESCRIPTION
**Pull request by vyarosh**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ networks:
 
 services:
   pmm-server:
-    image: ${PMM_SERVER_IMAGE:-perconalab/pmm-server:dev-latest}
+    image: ${PMM_SERVER_IMAGE:-perconalab/pmm-server-fb:PR-2296-eded043}
```
Comment (puneet0191) [inline docker-compose.yml:17]: any reason for using this FB?

**Review by puneet0191 — CHANGES_REQUESTED** [review]

File: tests/backup/mysql/scheduled_mysql_test.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+};
+
+let locationId;
+let serviceId;
+
+const mysqlServiceName = 'mysql-with-backup';
+const schedules = new DataTable(['cronExpression', 'name', 'frequency']);
+
+schedules.add(['30 8 * * *', 'schedule daily', 'At 08:30 AM']);
+schedules.add(['0 0 * * 2', 'schedule weekly', 'At 12:00 AM, only on Tuesday']);
```
Comment (puneet0191) [inline tests/backup/mysql/scheduled_mysql_test.js:18]: why do you have this data table if you are not using it?

**Review by puneet0191 — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (vyarosh) [reply docker-compose.yml]: missed this. Restored. Thank you for noticing!

File: tests/backup/mysql/scheduled_mysql_test.js (hunk 2 above)
Comment (vyarosh) [reply tests/backup/mysql/scheduled_mysql_test.js]: ah yeah! missed to clean up. Fixed

**Comment by puneet0191** [comment]
@vyarosh please fix conflicts on this one

**Comment by vyarosh** [comment]
merged. Conflict resolved it was due to renamed helper.

**Comment by puneet0191** [comment]
@vyarosh  linting errors on this PR now

**Comment by vyarosh** [comment]
Matrix: https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests-matrix/detail/pmm2-ui-tests-matrix/438/pipeline/

**Review by puneet0191 — APPROVED** [review]

**Review by yurkovychv — APPROVED** [review]

**Pull request merged**